tuxemon/states/choice/choice_item.py

```diff
@@ -14,6 +14,14 @@
 from tuxemon.menu.theme import get_theme
 
 ChoiceMenuGameObj = Callable[[], None]
+LENGTH_NAME_ITEM = 10
```

Comment:
This const seems to be unused.

Reply:
Same for the other files.

Reply:
Haven't checked if the other consts are used.

Reply:
it was the only one unused